cmd/create/breakglasscredential/cmd.go

```diff
@@ -0,0 +1,82 @@
+package breakglasscredential
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/breakglasscredential"
+	"github.com/openshift/rosa/pkg/externalauthprovider"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var breakGlassCredentialArgs *breakglasscredential.BreakGlassCredentialArgs
+
+const argsPrefix string = ""
+
+var Cmd = &cobra.Command{
+	Use:     "break-glass-credential",
+	Aliases: []string{"break-glass-credentials", "breakglasscredential", "breakglasscredentials"},
+	Short:   "Create a break glass credential for a cluster.",
+	Long:    "Create a break glass credential for a hosted control plane cluster with external OIDC.",
```

Comment:
```suggestion
	Long:    "Create a break glass credential for a hosted control plane cluster with external authentication enabled.",
```
To double check

Reply:
Thanks Andrea, will address those changes